### DISCUSSION TITLE
All checks have failed

### DISCUSSION THREAD

**Discussion by ukraine-sdg** [open-sdg/open-sdg#1500] [Q&A] [answered]
Hello!

Today, when I make changes in files in forders "data" and "translation", and then try to merge it, I get the message "All checks have failed". 

However, if I anyway push the bottom "Merge pull request", changes appear on the web-site.

Could you please explain, why I have a problem ("All checks have failed") during merging?

**Comment by LucyGwilliamAdmin** [comment, marked as answer]
@ukraine-sdg if you go into [the Action which fails](https://github.com/ukraine-sdg/data/runs/4454550526?check_suite_focus=true) and expand the 'Test and build data' section, you can see some more info about the error. You will find that there seems to be some issues in the Value column of the csv for 11.2.1

**Reply by ukraine-sdg** [reply]
Thank you!
